shared/p2putils/fork.go

```diff
@@ -42,3 +43,36 @@ func CreateForkDigest(
 	}
 	return digest, nil
 }
+
+// Fork given a target epoch and genesis validators root,
+// returns the active fork version in the node.
+func Fork(
```

Comment:
Is this code duplicated also in our p2p package? Maybe we can refactor to just use this shared method if that's the case

Reply:
i don't see that it is duplicated